### PR TITLE
Feature/orca 703 Documentation updates and cleanup

### PR DESCRIPTION
**Pull request by AndrewDorn**
## Summary of Changes

Added guide on running code locally and cleaned up some other docs.

Addresses [ORCA-703: Review documentation prior to exit](https://bugs.earthdata.nasa.gov/browse/ORCA-703) and [ORCA-670: Remove empty "Cookbook" pages from website](https://bugs.earthdata.nasa.gov/browse/ORCA-670)

## Changes

* Added local-debugging.md
* Various minor documentation fixes
* Removed redundant pages

## Type of change

- [x] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the necessary documentation (remove those that do not apply)
    - [x] Development documentation
    - [x] User documentation
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] My code has passed security scanning
    - [x] Snyk
    - [x] git-secrets

### CODE REVIEW

File: website/docs/developer/development-guide/code/setup-dev-env.md [hunk 1]
```diff
@@ -49,11 +49,15 @@ Lambdas should be built and tested within individual Virtual Environments.
...
+   pip install -r requirements.txt --trusted-host pypi.org --trusted-host files.pythonhosted.org
    ```
 1. Install additional development requirements.
    ```commandline
    pip install coverage
    pip install pylint
-   pip install -r requirements-dev.txt
-   ```
\ No newline at end of file
+   pip install -r requirements-dev.txt --trusted-host pypi.org --trusted-host files.pythonhosted.org
```
Comment (bhazuka) [inline website/docs/developer/development-guide/code/setup-dev-env.md:58]: See note from above on flags. Wondering if we should remove coverage and pylint? or possibly put a note to add all of the dev libraries that should be used like black, isort, pylint, coverage, etc in a note that they should be installed with a link to our template for run_test.sh? Thoughts?

**Review by bhazuka — CHANGES_REQUESTED** [review]
Overall looks good, added a couple of thoughts on the development notes for you to ponder.

File: website/docs/developer/development-guide/code/setup-dev-env.md [hunk 2]
```diff
@@ -49,11 +49,15 @@ Lambdas should be built and tested within individual Virtual Environments.
 1. With the VEnv activated, install requirements.
    ```commandline
    pip install boto3
-   pip install -r requirements.txt
+   pip install -r requirements.txt --trusted-host pypi.org --trusted-host files.pythonhosted.org
```
[the remark this answers is not in the document]
Comment (AndrewDorn) [reply website/docs/developer/development-guide/code/setup-dev-env.md:52]: You have it backwards. IT set up my computer to only recognize the VPN for these types of operations. Until I found this trick, I had to specifically connect to the "South Dakota Server" connection; even the NASA VPN doesn't work.

File: website/docs/developer/development-guide/code/setup-dev-env.md (hunk 1 above)
Comment (AndrewDorn) [reply website/docs/developer/development-guide/code/setup-dev-env.md:58]: `coverage` and `pylint` are mentioned in separate pages that focus on checking these things outside of our `run_tests.sh` script. If you want to refactor these pages, that is beyond the scope of the PR.

**Pull request merged**